MSXIV_Driver_Display/lib/constants/stdColors.dart

```diff
@@ -1,10 +1,34 @@
 import 'package:flutter/material.dart';
 
-class stdColors {
-  static const Color background = Color.fromRGBO(27, 27, 27, 1);
+class StdColors {
```

Comment:
any thoughts on renaming the file name to snake case? Based on what's already here I'm thinking it's camel case for class name; snake for file?

Reply:
> (this is not really a review, I can merge)
> 
> Future PRs -> can we include screenshots of current, implemented change versions in the PR description?

Sure we can do that!

also yep i think it makes sense to use snake case for file name, I can change that for the next PR since it's very minor change